diff --git a/node_modules/react-native-button/Button.js b/node_modules/react-native-button/Button.js
index fb7cf46..3004a12 100644
--- a/node_modules/react-native-button/Button.js
+++ b/node_modules/react-native-button/Button.js
@@ -9,6 +9,7 @@ import {
 } from 'react-native';
 
 import coalesceNonElementChildren from './coalesceNonElementChildren';
+import {TextPropTypes} from 'deprecated-react-native-prop-types'
 
 const systemButtonOpacity = 0.2;
 
@@ -16,15 +17,16 @@ export default class Button extends Component {
   static propTypes = {
     ...TouchableOpacity.propTypes,
     accessibilityLabel: PropTypes.string,
-    allowFontScaling: Text.propTypes.allowFontScaling,
+    allowFontScaling: TextPropTypes.allowFontScaling,
     containerStyle: ViewPropTypes.style,
     disabledContainerStyle: ViewPropTypes.style,
     disabled: PropTypes.bool,
-    style: Text.propTypes.style,
-    styleDisabled: Text.propTypes.style,
+    style: TextPropTypes.style,
+    styleDisabled: TextPropTypes.style,
     childGroupStyle: ViewPropTypes.style,
   };
 
+
   render() {
     let touchableProps = {
       activeOpacity: this._computeActiveOpacity(),
